docs/help/specifications-support/multiple-servers.md

```diff
@@ -0,0 +1,46 @@
+# Multiple Servers
+
+Both REST or Event-Driven APIs may run on different servers (production, staging, sandbox).
+
+And both [OpenAPI](https://spec.openapis.org/oas/v3.1.0#server-object) and [AsyncAPI](https://www.asyncapi.com/docs/reference/specification/v2.6.0#serverObject) provide field `servers`, where more than one server can be defined.
+
+With multiple servers support, it’s now possible to render these different servers in documentation.
+
+## Setting multiple servers
+
+Until now, only first server was extracted and displayed.
```

Comment:
```suggestion
```
I would remove this line: the doc describes the current state of the product, and disregards what was before.